### DISCUSSION TITLE
Pull Request for new user

### DISCUSSION THREAD

**Discussion by TrustBetOC** [community/community#76990] [Pull Requests]
### Select Topic Area

Question

### Body

Hi all! Im a new user and I have put in a pull request thats waiting to be approved by a maintainer. I got all green ticks for my work but its been 3 days now, is that normal or should I email support. Maybe i have missed something?
I'd appreciate any help.
Thank you

**Comment by jdtdm09** [comment]
1 - Check Project Guidelines: Make sure you've followed all the project's contribution guidelines. Some projects have specific requirements or processes that contributors need to adhere to. Review your changes against these guidelines to ensure everything is in order.

2 - Review Comments: If there are any comments or suggestions on your pull request, make sure to address them promptly. It's possible that the reviewer has requested changes or has additional questions that need clarification.

3 - Communication: If there's a project communication channel, like a mailing list or a chat platform, consider reaching out to the maintainers politely and inquire about the status of your pull request. You can ask if there's anything specific they would like you to address or if there's a reason for the delay.

4 - Be Patient: While it's understandable that you're eager to see your changes merged, keep in mind that maintainers might have a backlog of pull requests and other responsibilities. Patience is often key in open-source collaboration.

5 - Avoid Emailing Support Initially: Most open-source projects prefer communication through their established channels (like GitHub discussions or mailing lists) rather than direct emails to individual maintainers or support. Check the project's contribution guidelines for the recommended communication method.

**Reply by TrustBetOC** [reply]
Thank you so much for taking the time to reply. I appreciate it. I'll follow up on all your points.

**Reply by Akash1134** [reply]
Hi @jdtdm09 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
